.github/workflows/deploy.yml

```diff
@@ -0,0 +1,81 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ 'develop', 'master' ]
```

Comment:
Should run also on branches for which there is a requested PR (i.e. if merging from `branch-1` to `develop`, we should build and test `branch-1`)

Reply:
Ok.
Added, 
`pull_request: [ 'develop',  'master']`